src/search-schema/data/definitions/enums/organ_types.yaml

```diff
@@ -10,12 +10,16 @@
 
 AO:
   description: Aorta
+  iri: http://purl.obolibrary.org/obo/UBERON_0000947
 BL:
   description: Bladder
+  iri: http://purl.obolibrary.org/obo/UBERON_0018707

```

Comment:
Would the iri for ["Urinary Bladder"(https://www.ebi.ac.uk/ols/ontologies/uberon/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0001255) work better here? It is more specific than "Bladder organ"

Reply:
@pdblood @computationdoc I used the term for `Bladder` but @bhonick wants to know if `Urinary Bladder` is more acceptable/proper/correct. What do you think?

Reply:
And as an FYI, here are all the mappings we have from CCF-HRA Reference Organ to uberon term: <http://grlc.io/api-git/hubmapconsortium/ccf-grlc/subdir/ccf/ref-organ-terms>

Reply:
@bherr2 @bhonick I understand your suggestion but I need to know what is the action item. Should I keep `Bladder` or use `Urinary Bladder`? Whatever either/both of you think is best cc @pdblood

Reply:
@icaoberg I would recommend using the most specific descriptor for human anatomy. Therefore, `Urinary Bladder` would be the way to go.

Reply:
Resolved as per @bhonick suggestion.